BEIMA.Backend/MongoService/User.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace BEIMA.Backend.MongoService
+{
+    /// <summary>
+    /// Object representation of a last modified document in a User document
+    /// </summary>
+    public class UserLastModified
```

Comment:
Since this is used by multiple things, I think it would be worth it to put this in its own class file that all the database types can reference.

Reply:
Should we do this down the road? I know Kenny made some changes to Device.cs that does the same thing in his PR, so it might be a pain to try and coordinate changing over to a single class file right now. It might be easier to leave it as is right now and change them over all at once after everything is merged in.